src/nomad_measurements/general/__init__.py

```diff
@@ -0,0 +1,15 @@
+
+from nomad.config.models.plugins import SchemaPackageEntryPoint
+
+class GeneralSchemaPackageEntryPoint(SchemaPackageEntryPoint):
+
+    def load(self):
+        from nomad_measurements.general.schema import m_package
+
+        return m_package
+
+
+general_schema = GeneralSchemaPackageEntryPoint(
```

Comment:
As mentioned before this could be something like 
```suggestion
schema = GeneralSchemaPackageEntryPoint(
```